manifests/module.pp

```diff
@@ -44,11 +53,11 @@
   validate_absolute_path($::selinux::config::module_build_dir)
   validate_absolute_path($::selinux::refpolicy_makefile)
 
-  $module_dir = "${::selinux::config::module_build_dir}/${title}"
+  $module_dir = $::selinux::config::module_build_dir
```

Comment:
this and the above probably should be `$::selinux::module_build_dir` if the below is as well